twiml/voice/connect/virtualagent/virtualagent.3.x.js

```diff
@@ -0,0 +1,12 @@
+const VoiceResponse = require('twilio').twiml.VoiceResponse;
+
+const response = new VoiceResponse();
+const connect = response.connect({
+    action: 'https://myactionurl.com/twiml'
+});
+connect.virtualAgent({
+    connectorName: 'project',
+    statuscallback: 'https://mycallbackurl.com'
```

Comment:
That casing right?

Reply:
Tested this one and it's correct/works
```
❯ cat test.js
const VoiceResponse = require('twilio').twiml.VoiceResponse;

const response = new VoiceResponse();
const connect = response.connect({
    action: 'https://myactionurl.com/twiml'
});
connect.virtualAgent({
    connectorName: 'project',
    statuscallback: 'https://mycallbackurl.com'
});

console.log(response.toString());

~
❯ node test.js
<?xml version="1.0" encoding="UTF-8"?><Response><Connect action="https://myactionurl.com/twiml"><VirtualAgent connectorName="project" statuscallback="https://mycallbackurl.com"/></Connect></Response>
```

Reply:
oh nvm, javascript will allow anything. It should be camel case.